tests/Http/Responses/CreatedResponseTest.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\Tempest\Http\Responses;
+
+use PHPUnit\Framework\TestCase;
+use Tempest\Http\Responses\CreatedResponse;
+use Tempest\Http\Status;
+
+class CreatedResponseTest extends TestCase
+{
+    public function test_created_response()
+    {
+        $response = new CreatedResponse(json_encode(['foo' => 'bar']));
```

Comment:
@aidan-casey I think these response objects should convert arrays to json automatically, wdyt?

Nothing we should change in this PR, just a random thought I wanted to check up with you